packages/eslint-config-react-native-community/index.js

```diff
@@ -30,13 +31,39 @@ module.exports = {
     },
   },
 
-  overrides: {
-    files: ['**/__tests__/**/*.js', '**/*.spec.js', '**/*.test.js'],
-    env: {
-      jest: true,
-      'jest/globals': true,
+  overrides: [
+    {
+      files: ['*.js'],
+      parser: 'babel-eslint',
+      plugins: ['flowtype'],
+      rules: {
+        // Flow Plugin
+        // The following rules are made available via `eslint-plugin-flowtype`
+
+        'flowtype/define-flow-type': 1,
+        'flowtype/use-flow-type': 1,
+      },
     },
-  },
+    {
+      files: ['*.ts', '*.tsx'],
```

Comment:
```suggestion
      files: ['*.tsx?'],
```

To match jsx? 😄

Reply:
It's not the correct glob syntax for this afaik. from bash docs:

> If you specified something at the command line like "hd?" GNU/Linux would look for hda, hdb, hdc and every other letter/number between a-z, 0-9.

so `.tsx?` will look for `.tsxa`, `.tsxb` etc.

Reply:
sorry I meant 
files: ['*.ts?(x)'],
but it's not that important :)